### ISSUE TITLE
Open existing GIF in editor?

### ISSUE THREAD

**Issue by jakelauer** [gifox/tracker#292]
Gifox has an excellent editor, but I don't always need to do my editing immediately upon recording, and sometimes I'd like to edit a GIF that I didn't even make with Gifox. Is there a way to open files in the editor?

**Comment by lerinagalimova** [comment]
Hi @jakelauer , thanks for the compliment! The good news is that Gifox does support importing media for editing. 

![converting-importing-media-3](https://github.com/user-attachments/assets/c76d062f-1767-4ce8-8681-9a587d953650)

You can find more details about this feature in our documentation here: [Importing Media](https://gifox.app/docs/converting/#importing-media).

**Comment by jakelauer** [comment]
@lerinagalimova  yes, I saw that but it only works for video. I would like to import a gif. I tried it but it only imported the first frame

**Comment by iby** [comment]
Hey @jakelauer! The docs are a little out of date – apologies. It should work fine with GIFs too. The "first frame" problem is usually associated with how drag-and-drop is handled by different apps, it's a prehistoric issue, https://github.com/Gifox/Tracker/issues/24#issuecomment-314402304 – here's more details on that.

If you're dragging a GIF from a browser or some other app, try saving it first and doing it from Finder instead. In fact, you can drag-and-drop GIFs from Gifox like that just to check it's working. 

If that doesn't apply and the issue is with something else, would be great if you can share a GIF that causes troubles and I'll have a deeper look into that.
